pages/finance.tsx

```diff
@@ -20,10 +20,10 @@ function Finance() {
   const [values, setValues] = useState({
     dateHeaderCount: 8,
     startDate: dateBeforeDays(3),
-    endDate: dateAfterDays(4),
+    endDate: dateAfterDays(3),
```

Comment:
return the number of dates before and after fetched from a helper method. Later on that method will consider the available width to figure out the number of dates to display

Reply:
would want to do this seperately.